server/poll/poll.go

```diff
@@ -75,6 +85,36 @@ func NewPoll(creator, question string, answerOptions, settings []string) (*Poll,
 	return &p, nil
 }
 
+// ParseVotesSetting parses and sets a setting for votes ("--votes=X")
+func (p *Poll) ParseVotesSetting(s string) *ErrorMessage {
+	e := votesSettingPattern.FindStringSubmatch(s)
+	if len(e) != 2 {
+		return &ErrorMessage{
+			Message: &i18n.Message{
+				ID:    "poll.newPoll.votesettings.unexpectedError",
+				Other: "Unexpected error happens when parsing {{.Setting}}",
+			},
+			Data: map[string]interface{}{
+				"Setting": s,
+			},
+		}
+	}
+	i, err := strconv.Atoi(e[1])
+	if err != nil || i <= 0 || i > len(p.AnswerOptions) {
+		return &ErrorMessage{
+			Message: &i18n.Message{
+				ID:    "poll.newPoll.votesettings.invalidSetting",
+				Other: "In votes=X, X must be a positive number and less than the number of answer options. {{.Setting}}",
```

Comment:
That error message looks weird in the modal. Could it me more generic and the command handlers adds it's specific information itself?
![Screenshot from 2020-05-29 10-56-56](https://user-images.githubusercontent.com/16541325/83241468-2ae6e800-a19b-11ea-8b21-c814ee80a33c.png)

Reply:
X is a temporary number, and the value is not embedded.
It's certainly weird to see it in dialog. I'm going to amend it to say, `The number of votes must be a positive number and less than or equal to the number of options. You specified "votes=50."`